keyboards/ergodox_ez/config.h

```diff
@@ -114,5 +114,6 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 //#define NO_ACTION_MACRO
 //#define NO_ACTION_FUNCTION
 //#define DEBUG_MATRIX_SCAN_RATE
+#define QMK_KEYS_PER_SCAN 8
```

Comment:
Could we put this in a specific keymap and not in the overriding config.h? I am concerned of potential effects on our many users.